joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -325,6 +360,37 @@ JointTrajectoryController::halt()
   set_op_mode(hardware_interface::OperationMode::ACTIVE);
 }
 
+void JointTrajectoryController::publish_state()
+{
+  if (state_publisher_period_.seconds() <= 0.0) {
+    return;
+  }
+
+  if (lifecycle_node_->now() < (last_state_publish_time_ + state_publisher_period_)) {
+    return;
+  }
+
+  if (state_publisher_ && state_publisher_->trylock()) {
+    last_state_publish_time_ = lifecycle_node_->now();
+
+    state_publisher_->msg_.header.stamp = last_state_publish_time_;
+    // TODO(ddengster): The following code is ported from ros_controllers.
+    // Port it when we're ready to put in the other trajectory code.
```

Comment:
don't push commented code blocks please

Reply:
I feel we had this comment here already. I'd recommend opening a follow up issue for it.

Reply:
removed in 99c036d